test/schemas/modules/unsupported.ext

```diff
@@ -1 +1 @@
-This file should not be loaded
+// This file should not be loaded
```

Comment:
Is it really needed?

Reply:
Yes, because linter fails while parsing this file